concordium-sdk/src/main/java/com/concordium/sdk/responses/peerStats/PeerStatistics.java

```diff
@@ -0,0 +1,43 @@
+package com.concordium.sdk.responses.peerStats;
+
+import com.google.common.collect.ImmutableList;
+import concordium.ConcordiumP2PRpc;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.val;
+
+import java.util.stream.Collectors;
+
+@Builder
+@Getter
+public class PeerStatistics {
```

Comment:
Please add a `toString` for convenience.

Reply:
done